widgets/alternate/plus.svg

```diff
@@ -2,66 +2,66 @@
 <!-- Created with Inkscape (http://www.inkscape.org/) -->
 
 <svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Comment:
No need to change indentation on svg files, please revert changes related to this.